typescript/README.md

```diff
@@ -0,0 +1,9 @@
+name: TypeScript
+
+description: A typed superset of JavaScript.
```

Comment:
While this is true, both `typed` and `superset` might be confusing as part of the description. Maybe `Extend JavaScript syntax to make it more resilient and safer`

Reply:
Thats now TS describes itself

> TypeScript is a typed superset of JavaScript that compiles to plain JavaScript.

But this is their og desc 

> TypeScript brings you optional static type-checking along with the latest ECMAScript features.

Reply:
Maybe we can go with their og desc

Reply:
![image](https://user-images.githubusercontent.com/19802557/80382236-f3ce9f80-8899-11ea-8138-25c74baf86ba.png)

too long, we should stick to only 2 lines of text so more topics get visibility. also, I feel like the og description is very technical. beginners would be really confused

Reply:
I like their own description

> TypeScript is a typed superset of JavaScript that compiles to plain JavaScript.